AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/TaskController.cs

```diff
@@ -40,7 +40,10 @@ public IActionResult Create(int activityId)
                 ActivityName = activity.Name,
                 CampaignId = activity.CampaignId,
                 CampaignName = activity.Campaign.Name,
-                TenantId = activity.Campaign.ManagingTenantId
+                TenantId = activity.Campaign.ManagingTenantId,
+                TimeZoneId = activity.Campaign.TimeZoneId,
+                StartDateTime = activity.StartDateTime.LocalDateTime,

```

Comment:
Don't convert to LocalDateTime here. You should set the StartDateTime to activity.StartDateTime only. The time is stored and displayed in the timezone corresponding to that Campaign.